projects/appearances/protos/ArtificialGrass.proto

```diff
@@ -0,0 +1,32 @@
+#VRML_SIM R2019b utf8
+# license: Apache License 2.0
+# license url: http://www.apache.org/licenses/LICENSE-2.0
+# An artificial grass material.
+
+PROTO ArtificialGrass [
```

Comment:
I would like to add more grass texture soon. I think it would be better to rename this PROTO simply to Grass, and add a `type` field set to `artificial` later.